tests/translated_table_model_tester.ts

```diff
@@ -11,6 +11,87 @@ function getTestTable() {
   return matrix;
 }
 
+function getLongTestTable() {
+  const matrix = new ArrayTableModel();
+  matrix.push([0]);
+  matrix.push([1]);
+  matrix.push([2]);
+  matrix.push([3]);
+  matrix.push([4]);
+  matrix.push([5]);
+  matrix.push([6]);
+  matrix.push([7]);
+  return matrix;
+}
+
+function shuffleRows(table: TranslatedTableModel) {
+  table.moveRow(7, 0);
+  table.moveRow(6, 1);
+  table.moveRow(5, 2);
+  table.moveRow(3, 4);
+}
+
+class TableMatcher extends Matcher<TableModel | (() => any)> {
+  toEqualCells(expected: any[][]) {
```

Comment:
Unit test functionality must still have all the proper type annotations and access specifiers.